calcom/cal.com
# fix: Added relevant colors to preset and replaced with tailwind colors

## What does this PR do?

I have added relevant colors to the tailwind preset which weren't present. Then I have updated the tailwind default colors to custom colors. Also fixed some focus state issues.

Fixes #9507 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Since I have updated the storybook too, you can go to the checkbox story to check the new added colors.

## Mandatory Tasks
